### ISSUE TITLE
IdsEditor: e2e test failures

### ISSUE THREAD

**Issue by EdwardCoyle** [infor-design/enterprise-wc#1193]
**Describe the bug**
Currently on `main` branch, IdsEditor is failing several e2e tests that are causing other branches to fail on merge:

<img width="736" alt="Screen Shot 2023-03-20 at 10 55 11 AM" src="https://user-images.githubusercontent.com/3249601/226382257-32c75d56-c56b-4449-ae86-2a77925e894f.png">

**To Reproduce**
- Pull `main`
- Run `npm run test -- ids-editor-e2e`.
- See test failures

**Expected behavior**

**Version**
`main` branch at 1.0.0-beta8

**Comment by tmcconechy** [comment]
@EdwardCoyle i dont see this on my end but make sure to `npm run build` for a clean build since if you built with a livereload part might not be there?

```
git pull origin main
git pull
npm prune && npm i
npm run build
npm run test -- ids-editor-e2e
```

```
tim.mcconechy@USROMTMCCONEC02 enterprise-wc4 % npm run test -- ids-editor-e2e

> ids-enterprise-wc@1.0.0-beta.7 test
> PERCY_LOGLEVEL=silent npx jest --coverage --runInBand --detectOpenHandles --forceExit ids-editor-e2e

Dev server listening on port 4444
 PASS  test/ids-editor/ids-editor-e2e-test.ts (6.356 s)
  Ids Editor e2e Tests
    ✓ should not have errors (9 ms)
    ✓ should pass Axe accessibility tests (1925 ms)
    ✓ should make text bold, italic, underline, strikethrough and clearformatting (43 ms)
    ✓ should toggle blockquote (6 ms)
    ✓ should set text-align left and center (5 ms)
    ✓ should set text-align left and right (4 ms)
    ✓ should toggle ordered list (9 ms)
    ✓ should toggle unordered list (7 ms)
    ✓ should toggle header elements (49 ms)
    ✓ should toggle hyperlink (17 ms)
    ✓ should be able to activate buttons on selection change (6 ms)
    ✓ should be able to use source formatter (21 ms)
    ✓ should toggle fore color (43 ms)
    ✓ should be able to insert image (61 ms)
    ○ skipped should not have memory leaks
```

**Comment by EdwardCoyle** [comment]
Seems to be an issue with just my branch on second glance.  Will fix on #1174
